forum/apps.py

```diff
@@ -33,4 +33,5 @@ def ready(self) -> None:
         """
         Import Signals.
         """
-        import forum.signals  # pylint: disable=import-outside-toplevel, unused-import
+        import forum.api  # pylint: disable=import-outside-toplevel, unused-import
```

Comment:
What is this for?

Reply:
I'll just remove it as now the code is working with latest changes. 
I was facing issue while detecting forum v2 in edx-platform when i import something from edx-platform.